app/tailwind.css

```diff
@@ -2,6 +2,32 @@
 @tailwind components;
 @tailwind utilities;
 
+:root {
+  --vd-beige-200: 247 237 232;
```

Comment:
did you have to paste each hex into a converter or was there a shortcut?

Reply:
I used a shortcut — I asked [cursor](https://cursor.sh) to convert it